### DISCUSSION TITLE
Triage parsing

### DISCUSSION THREAD

**Discussion by du27** [knez/defender-dump#2] [Ideas]
Hi,

This tool would be super handy if it could be used against a triage image. What directories does it process in order to get an output. I've tried copying the entire ProgramData/Microsoft/Windows Defender directory. But that throws an error when trying run
Traceback (most recent call last):
  File "C:\Users\...\Desktop\defender-dump.py", line 182, in <module>
    main(parser.parse_args())
  File "C:\Users\...\Desktop\defender-dump.py", line 165, in main
    detection_max_len = max([len(x[2]) for x in entries])
ValueError: max() arg is an empty sequence

**Comment by du27** [comment]
> Hi,
> 
> This tool would be super handy if it could be used against a triage image. What directories does it process in order to get an output. I've tried copying the entire ProgramData/Microsoft/Windows Defender directory. But that throws an error when trying run Traceback (most recent call last): File "C:\Users...\Desktop\defender-dump.py", line 182, in main(parser.parse_args()) File "C:\Users...\Desktop\defender-dump.py", line 165, in main detection_max_len = max([len(x[2]) for x in entries]) ValueError: max() arg is an empty sequence

Nevermind, it does work. I was missing files.

**Reply by knez** [reply]
Hi,

Yes, the base folder is ProgramData/Microsoft/Windows Defender/Quarantine. Although the script only needs two subfolders Quarantine/Entries and Quarantine/ResourceData for it to work correctly.
